### ISSUE TITLE
cabal new-install TARGET generates dead link.

### ISSUE THREAD

**Issue by Magicloud** [haskell/cabal#5649]
Following command exited 1. Made dead symlinks in ~/.cabal/bin. ~/.cabal/store/ghc-8.6.1 does not contain cabal-src or mega-sdist, but many other folders for those libs (seems like).

```
$ cabal new-install cabal-src -j2                       
Resolving dependencies...
Build profile: -w ghc-8.6.1 -O1
In order, the following will be built (use -v for more details):
 - base-compat-0.10.5 (lib) (requires download & build)
 - base-orphans-0.8 (lib) (requires download & build)
 - basement-0.0.8 (lib) (requires download & build)
 - blaze-builder-0.4.1.0 (lib) (requires download & build)
 - byteable-0.1.1 (lib) (requires download & build)
 - cereal-0.5.7.0 (lib) (requires download & build)
 - data-default-class-0.1.2.0 (lib:data-default-class) (requires download & build)
 - dlist-0.8.0.5 (lib) (requires download & build)
 - hashable-1.2.7.0 (lib) (requires download & build)
 - hourglass-0.2.12 (lib) (requires download & build)
 - integer-logarithms-1.0.2.2 (lib) (requires download & build)
 - mime-types-0.1.0.8 (lib) (requires download & build)
 - network-2.8.0.0 (lib:network) (requires download & build)
 - network-uri-2.6.1.0 (lib) (requires download & build)
 - primitive-0.6.4.0 (lib) (requires download & build)
 - random-1.1 (lib) (requires download & build)
 - semigroups-0.18.5 (lib) (requires download & build)
 - split-0.2.3.3 (lib) (requires download & build)
 - system-filepath-0.4.14 (lib:system-filepath) (requires download & build)
 - tagged-0.8.6 (lib) (requires download & build)
 - tar-0.5.1.0 (lib) (requires download & build)
 - th-abstraction-0.2.8.0 (lib) (requires download & build)
 - time-locale-compat-0.1.1.5 (lib) (requires download & build)
 - transformers-compat-0.6.2 (lib) (requires download & build)
 - unix-compat-0.5.1 (lib:unix-compat) (requires download & build)
 - unliftio-core-0.1.2.0 (lib) (requires download & build)
 - zlib-0.6.2 (lib) (requires download & build)
 - memory-0.14.18 (lib) (requires download & build)
 - cookie-0.4.4 (lib) (requires download & build)
 - unordered-containers-0.2.9.0 (lib) (requires download & build)
 - case-insensitive-1.2.0.11 (lib) (requires download & build)
 - async-2.2.1 (lib) (requires download & build)
 - socks-0.5.6 (lib) (requires download & build)
 - vector-0.12.0.1 (lib) (requires download & build)
 - scientific-0.3.6.2 (lib) (requires download & build)
 - uuid-types-1.0.3 (lib) (requires download & build)
 - system-fileio-0.3.16.4 (lib) (requires download & build)
 - transformers-base-0.4.5.2 (lib) (requires download & build)
 - exceptions-0.10.0 (lib) (requires download & build)
 - constraints-0.10.1 (lib) (requires download & build)
 - pem-0.2.4 (lib) (requires download & build)
 - cryptonite-0.25 (lib) (requires download & build)
 - asn1-types-0.3.2 (lib:asn1-types) (requires download & build)
 - http-types-0.12.2 (lib) (requires download & build)
 - typed-process-0.2.3.0 (lib) (requires download & build)
 - streaming-commons-0.2.1.0 (lib) (requires download & build)
 - vector-algorithms-0.8.0.1 (lib) (requires download & build)
 - attoparsec-0.13.2.2 (lib) (requires download & build)
 - monad-control-1.0.2.3 (lib:monad-control) (requires download & build)
 - resourcet-1.2.2 (lib) (requires download & build)
 - asn1-encoding-0.9.5 (lib) (requires download & build)
 - http-client-0.5.13.1 (lib) (requires download & build)
 - mono-traversable-1.0.9.0 (lib) (requires download & build)
 - aeson-1.4.1.0 (lib) (requires download & build)
 - lifted-base-0.2.3.12 (lib) (requires download & build)
 - asn1-parse-0.9.4 (lib:asn1-parse) (requires download & build)
 - conduit-1.3.1 (lib) (requires download & build)
 - lifted-async-0.10.0.3 (lib) (requires download & build)
 - enclosed-exceptions-1.0.3 (lib) (requires download & build)
 - x509-1.7.4 (lib) (requires download & build)
 - conduit-extra-1.3.0 (lib) (requires download & build)
 - shelly-1.8.1 (lib) (requires download & build)
 - x509-store-1.6.6 (lib) (requires download & build)
 - x509-validation-1.6.10 (lib) (requires download & build)
 - x509-system-1.6.6 (lib) (requires download & build)
 - tls-1.4.1 (lib) (requires download & build)
 - connection-0.2.8 (lib:connection) (requires download & build)
 - http-client-tls-0.3.5.3 (lib) (requires download & build)
 - http-conduit-2.3.2 (lib) (requires download & build)
 - cabal-src-0.3.0.2 (exe:cabal-src-install, exe:mega-sdist) (requires download & build)
Downloading  base-orphans-0.8
Downloaded   base-orphans-0.8
Downloading  basement-0.0.8
Starting     base-orphans-0.8 (lib)
Building     base-orphans-0.8 (lib)
Installing   base-orphans-0.8 (lib)
Completed    base-orphans-0.8 (lib)
Downloaded   basement-0.0.8
Downloading  data-default-class-0.1.2.0
Starting     basement-0.0.8 (lib)
Building     basement-0.0.8 (lib)
Downloaded   data-default-class-0.1.2.0
Downloading  semigroups-0.18.5
Downloaded   semigroups-0.18.5
Downloading  split-0.2.3.3
Downloaded   split-0.2.3.3
Downloading  primitive-0.6.4.0
Downloaded   primitive-0.6.4.0
Downloading  transformers-compat-0.6.2
Downloaded   transformers-compat-0.6.2
Downloading  unliftio-core-0.1.2.0
Downloaded   unliftio-core-0.1.2.0
Downloading  byteable-0.1.1
Downloaded   byteable-0.1.1
Downloading  zlib-0.6.2
Downloaded   zlib-0.6.2
Downloading  cereal-0.5.7.0
Downloaded   cereal-0.5.7.0
Downloading  dlist-0.8.0.5
Downloaded   dlist-0.8.0.5
Downloading  hourglass-0.2.12
Downloaded   hourglass-0.2.12
Downloading  memory-0.14.18
Downloaded   memory-0.14.18
Downloading  asn1-types-0.3.2
Downloaded   asn1-types-0.3.2
Downloading  asn1-encoding-0.9.5
Downloaded   asn1-encoding-0.9.5
Downloading  asn1-parse-0.9.4
Downloaded   asn1-parse-0.9.4
Downloading  cryptonite-0.25
Downloaded   cryptonite-0.25
Downloading  pem-0.2.4
Downloaded   pem-0.2.4
Downloading  x509-1.7.4
Downloaded   x509-1.7.4
Downloading  th-abstraction-0.2.8.0
Downloaded   th-abstraction-0.2.8.0
Downloading  tagged-0.8.6
Downloaded   tagged-0.8.6
Downloading  random-1.1
Downloaded   random-1.1
Downloading  time-locale-compat-0.1.1.5
Downloaded   time-locale-compat-0.1.1.5
Downloading  base-compat-0.10.5
Downloaded   base-compat-0.10.5
Downloading  x509-store-1.6.6
Starting     base-compat-0.10.5 (lib)
Building     base-compat-0.10.5 (lib)
Downloaded   x509-store-1.6.6
Downloading  x509-validation-1.6.10
Downloaded   x509-validation-1.6.10
Downloading  network-2.8.0.0
Installing   base-compat-0.10.5 (lib)
Installing   basement-0.0.8 (lib)
Completed    base-compat-0.10.5 (lib)
Completed    basement-0.0.8 (lib)
Starting     byteable-0.1.1 (lib)
Building     byteable-0.1.1 (lib)
Installing   byteable-0.1.1 (lib)
Completed    byteable-0.1.1 (lib)
Starting     cereal-0.5.7.0 (lib)
Building     cereal-0.5.7.0 (lib)
Downloaded   network-2.8.0.0
Downloading  socks-0.5.6
Downloaded   socks-0.5.6
Downloading  x509-system-1.6.6
Downloaded   x509-system-1.6.6
Downloading  unix-compat-0.5.1
Downloaded   unix-compat-0.5.1
Downloading  vector-0.12.0.1
Downloaded   vector-0.12.0.1
Downloading  vector-algorithms-0.8.0.1
Downloaded   vector-algorithms-0.8.0.1
Downloading  integer-logarithms-1.0.2.2
Installing   cereal-0.5.7.0 (lib)
Completed    cereal-0.5.7.0 (lib)
Starting     data-default-class-0.1.2.0 (all, legacy fallback)
Building     data-default-class-0.1.2.0 (all, legacy fallback)
Installing   data-default-class-0.1.2.0 (all, legacy fallback)
Completed    data-default-class-0.1.2.0 (all, legacy fallback)
Starting     dlist-0.8.0.5 (lib)
Downloaded   integer-logarithms-1.0.2.2
Downloading  exceptions-0.10.0
Building     dlist-0.8.0.5 (lib)
Downloaded   exceptions-0.10.0
Downloading  resourcet-1.2.2
Installing   dlist-0.8.0.5 (lib)
Downloaded   resourcet-1.2.2
Downloading  transformers-base-0.4.5.2
Completed    dlist-0.8.0.5 (lib)
Downloaded   transformers-base-0.4.5.2
Downloading  monad-control-1.0.2.3
Downloaded   monad-control-1.0.2.3
Downloading  lifted-base-0.2.3.12
Downloaded   lifted-base-0.2.3.12
Downloading  enclosed-exceptions-1.0.3
Downloaded   enclosed-exceptions-1.0.3
Downloading  tar-0.5.1.0
Downloaded   tar-0.5.1.0
Downloading  blaze-builder-0.4.1.0
Downloaded   blaze-builder-0.4.1.0
Downloading  cookie-0.4.4
Starting     blaze-builder-0.4.1.0 (lib)
Building     blaze-builder-0.4.1.0 (lib)
Downloaded   cookie-0.4.4
Downloading  hashable-1.2.7.0
Downloaded   hashable-1.2.7.0
Downloading  async-2.2.1
Starting     hashable-1.2.7.0 (lib)
Installing   blaze-builder-0.4.1.0 (lib)
Completed    blaze-builder-0.4.1.0 (lib)
Starting     hourglass-0.2.12 (lib)
Building     hashable-1.2.7.0 (lib)
Building     hourglass-0.2.12 (lib)
Downloaded   async-2.2.1
Downloading  tls-1.4.1
Installing   hashable-1.2.7.0 (lib)
Completed    hashable-1.2.7.0 (lib)
Starting     integer-logarithms-1.0.2.2 (lib)
Building     integer-logarithms-1.0.2.2 (lib)
Installing   integer-logarithms-1.0.2.2 (lib)
Completed    integer-logarithms-1.0.2.2 (lib)
Downloaded   tls-1.4.1
Downloading  connection-0.2.8
Downloaded   connection-0.2.8
Downloading  typed-process-0.2.3.0
Installing   hourglass-0.2.12 (lib)
Completed    hourglass-0.2.12 (lib)
Starting     network-2.8.0.0 (all, legacy fallback)
Downloaded   typed-process-0.2.3.0
Downloading  case-insensitive-1.2.0.11
Downloaded   case-insensitive-1.2.0.11
Downloading  constraints-0.10.1
Downloaded   constraints-0.10.1
Downloading  lifted-async-0.10.0.3
Downloaded   lifted-async-0.10.0.3
Downloading  unordered-containers-0.2.9.0
Downloaded   unordered-containers-0.2.9.0
Downloading  http-types-0.12.2
Building     network-2.8.0.0 (all, legacy fallback)
Downloaded   http-types-0.12.2
Downloading  mime-types-0.1.0.8
Downloaded   mime-types-0.1.0.8
Downloading  mono-traversable-1.0.9.0
Starting     mime-types-0.1.0.8 (lib)
Building     mime-types-0.1.0.8 (lib)
Downloaded   mono-traversable-1.0.9.0
Downloading  conduit-1.3.1
Downloaded   conduit-1.3.1
Downloading  network-uri-2.6.1.0
Downloaded   network-uri-2.6.1.0
Downloading  scientific-0.3.6.2
Downloaded   scientific-0.3.6.2
Downloading  attoparsec-0.13.2.2
Downloaded   attoparsec-0.13.2.2
Downloading  streaming-commons-0.2.1.0
Downloaded   streaming-commons-0.2.1.0
Downloading  conduit-extra-1.3.0
Downloaded   conduit-extra-1.3.0
Downloading  http-client-0.5.13.1
Downloaded   http-client-0.5.13.1
Downloading  http-client-tls-0.3.5.3
Downloaded   http-client-tls-0.3.5.3
Downloading  uuid-types-1.0.3
Downloaded   uuid-types-1.0.3
Downloading  aeson-1.4.1.0
Installing   mime-types-0.1.0.8 (lib)
Completed    mime-types-0.1.0.8 (lib)
Starting     network-uri-2.6.1.0 (lib)
Building     network-uri-2.6.1.0 (lib)
Installing   network-2.8.0.0 (all, legacy fallback)
Downloaded   aeson-1.4.1.0
Downloading  http-conduit-2.3.2
Completed    network-2.8.0.0 (all, legacy fallback)
Starting     primitive-0.6.4.0 (lib)
Building     primitive-0.6.4.0 (lib)
Downloaded   http-conduit-2.3.2
Downloading  system-filepath-0.4.14
Downloaded   system-filepath-0.4.14
Downloading  system-fileio-0.3.16.4
Downloaded   system-fileio-0.3.16.4
Downloading  shelly-1.8.1
Installing   network-uri-2.6.1.0 (lib)
Completed    network-uri-2.6.1.0 (lib)
Starting     random-1.1 (lib)
Building     random-1.1 (lib)
Downloaded   shelly-1.8.1
Downloading  cabal-src-0.3.0.2
Downloaded   cabal-src-0.3.0.2
Installing   random-1.1 (lib)
Completed    random-1.1 (lib)
Starting     semigroups-0.18.5 (lib)
Installing   primitive-0.6.4.0 (lib)
Completed    primitive-0.6.4.0 (lib)
Starting     split-0.2.3.3 (lib)
Building     semigroups-0.18.5 (lib)
Building     split-0.2.3.3 (lib)
Installing   semigroups-0.18.5 (lib)
Completed    semigroups-0.18.5 (lib)
Starting     system-filepath-0.4.14 (all, legacy fallback)
Installing   split-0.2.3.3 (lib)
Completed    split-0.2.3.3 (lib)
Starting     tagged-0.8.6 (lib)
Building     tagged-0.8.6 (lib)
Building     system-filepath-0.4.14 (all, legacy fallback)
Installing   tagged-0.8.6 (lib)
Completed    tagged-0.8.6 (lib)
Starting     tar-0.5.1.0 (lib)
Building     tar-0.5.1.0 (lib)
Installing   system-filepath-0.4.14 (all, legacy fallback)
Completed    system-filepath-0.4.14 (all, legacy fallback)
Starting     th-abstraction-0.2.8.0 (lib)
Building     th-abstraction-0.2.8.0 (lib)
Installing   tar-0.5.1.0 (lib)
Completed    tar-0.5.1.0 (lib)
Starting     time-locale-compat-0.1.1.5 (lib)
Building     time-locale-compat-0.1.1.5 (lib)
Installing   time-locale-compat-0.1.1.5 (lib)
Completed    time-locale-compat-0.1.1.5 (lib)
Starting     transformers-compat-0.6.2 (lib)
Building     transformers-compat-0.6.2 (lib)
Installing   th-abstraction-0.2.8.0 (lib)
Completed    th-abstraction-0.2.8.0 (lib)
Starting     unix-compat-0.5.1 (all, legacy fallback)
Installing   transformers-compat-0.6.2 (lib)
Building     unix-compat-0.5.1 (all, legacy fallback)
Completed    transformers-compat-0.6.2 (lib)
Starting     unliftio-core-0.1.2.0 (lib)
Building     unliftio-core-0.1.2.0 (lib)
Installing   unix-compat-0.5.1 (all, legacy fallback)
Installing   unliftio-core-0.1.2.0 (lib)
Completed    unix-compat-0.5.1 (all, legacy fallback)
Starting     zlib-0.6.2 (lib)
Completed    unliftio-core-0.1.2.0 (lib)
Starting     memory-0.14.18 (lib)
Building     zlib-0.6.2 (lib)
Building     memory-0.14.18 (lib)
Installing   zlib-0.6.2 (lib)
Completed    zlib-0.6.2 (lib)
Starting     cookie-0.4.4 (lib)
Building     cookie-0.4.4 (lib)
Installing   memory-0.14.18 (lib)
Completed    memory-0.14.18 (lib)
Starting     unordered-containers-0.2.9.0 (lib)
Building     unordered-containers-0.2.9.0 (lib)
Installing   cookie-0.4.4 (lib)
Completed    cookie-0.4.4 (lib)
Starting     case-insensitive-1.2.0.11 (lib)
Building     case-insensitive-1.2.0.11 (lib)
Installing   case-insensitive-1.2.0.11 (lib)
Completed    case-insensitive-1.2.0.11 (lib)
Starting     async-2.2.1 (lib)
Building     async-2.2.1 (lib)
Installing   async-2.2.1 (lib)
Completed    async-2.2.1 (lib)
Starting     socks-0.5.6 (lib)
Building     socks-0.5.6 (lib)
Installing   unordered-containers-0.2.9.0 (lib)
Installing   socks-0.5.6 (lib)
Completed    socks-0.5.6 (lib)
Starting     uuid-types-1.0.3 (lib)
Completed    unordered-containers-0.2.9.0 (lib)
Starting     vector-0.12.0.1 (lib)
Building     uuid-types-1.0.3 (lib)
Building     vector-0.12.0.1 (lib)
Installing   uuid-types-1.0.3 (lib)
Completed    uuid-types-1.0.3 (lib)
Starting     scientific-0.3.6.2 (lib)
Building     scientific-0.3.6.2 (lib)
Installing   scientific-0.3.6.2 (lib)
Completed    scientific-0.3.6.2 (lib)
Starting     system-fileio-0.3.16.4 (lib)
Building     system-fileio-0.3.16.4 (lib)
Installing   system-fileio-0.3.16.4 (lib)
Completed    system-fileio-0.3.16.4 (lib)
Starting     transformers-base-0.4.5.2 (lib)
Building     transformers-base-0.4.5.2 (lib)
Installing   transformers-base-0.4.5.2 (lib)
Completed    transformers-base-0.4.5.2 (lib)
Starting     exceptions-0.10.0 (lib)
Building     exceptions-0.10.0 (lib)
Installing   exceptions-0.10.0 (lib)
Completed    exceptions-0.10.0 (lib)
Starting     constraints-0.10.1 (lib)
Building     constraints-0.10.1 (lib)
Installing   constraints-0.10.1 (lib)
Completed    constraints-0.10.1 (lib)
Starting     pem-0.2.4 (lib)
Building     pem-0.2.4 (lib)
Installing   pem-0.2.4 (lib)
Completed    pem-0.2.4 (lib)
Starting     cryptonite-0.25 (lib)
Building     cryptonite-0.25 (lib)
Installing   vector-0.12.0.1 (lib)
Completed    vector-0.12.0.1 (lib)
Starting     asn1-types-0.3.2 (all, legacy fallback)
Building     asn1-types-0.3.2 (all, legacy fallback)
Installing   asn1-types-0.3.2 (all, legacy fallback)
Completed    asn1-types-0.3.2 (all, legacy fallback)
Starting     http-types-0.12.2 (lib)
Building     http-types-0.12.2 (lib)
Installing   http-types-0.12.2 (lib)
Completed    http-types-0.12.2 (lib)
Starting     typed-process-0.2.3.0 (lib)
Building     typed-process-0.2.3.0 (lib)
Installing   typed-process-0.2.3.0 (lib)
Completed    typed-process-0.2.3.0 (lib)
Starting     streaming-commons-0.2.1.0 (lib)
Building     streaming-commons-0.2.1.0 (lib)
Installing   cryptonite-0.25 (lib)
Completed    cryptonite-0.25 (lib)
Starting     attoparsec-0.13.2.2 (lib)
Installing   streaming-commons-0.2.1.0 (lib)
Building     attoparsec-0.13.2.2 (lib)
Completed    streaming-commons-0.2.1.0 (lib)
Starting     monad-control-1.0.2.3 (all, legacy fallback)
Building     monad-control-1.0.2.3 (all, legacy fallback)
Installing   monad-control-1.0.2.3 (all, legacy fallback)
Completed    monad-control-1.0.2.3 (all, legacy fallback)
Starting     resourcet-1.2.2 (lib)
Building     resourcet-1.2.2 (lib)
Installing   resourcet-1.2.2 (lib)
Completed    resourcet-1.2.2 (lib)
Starting     vector-algorithms-0.8.0.1 (lib)
Building     vector-algorithms-0.8.0.1 (lib)
Installing   vector-algorithms-0.8.0.1 (lib)
Completed    vector-algorithms-0.8.0.1 (lib)
Starting     asn1-encoding-0.9.5 (lib)
Building     asn1-encoding-0.9.5 (lib)
Installing   attoparsec-0.13.2.2 (lib)
Completed    attoparsec-0.13.2.2 (lib)
Starting     http-client-0.5.13.1 (lib)
Installing   asn1-encoding-0.9.5 (lib)
Completed    asn1-encoding-0.9.5 (lib)
Starting     lifted-base-0.2.3.12 (lib)
Building     http-client-0.5.13.1 (lib)
Building     lifted-base-0.2.3.12 (lib)
Installing   lifted-base-0.2.3.12 (lib)
Completed    lifted-base-0.2.3.12 (lib)
Starting     mono-traversable-1.0.9.0 (lib)
Building     mono-traversable-1.0.9.0 (lib)
Installing   http-client-0.5.13.1 (lib)
Completed    http-client-0.5.13.1 (lib)
Starting     aeson-1.4.1.0 (lib)
Building     aeson-1.4.1.0 (lib)
Installing   mono-traversable-1.0.9.0 (lib)
Completed    mono-traversable-1.0.9.0 (lib)
Starting     asn1-parse-0.9.4 (all, legacy fallback)
Building     asn1-parse-0.9.4 (all, legacy fallback)
Installing   asn1-parse-0.9.4 (all, legacy fallback)
Completed    asn1-parse-0.9.4 (all, legacy fallback)
Starting     lifted-async-0.10.0.3 (lib)
Building     lifted-async-0.10.0.3 (lib)
Installing   lifted-async-0.10.0.3 (lib)
Completed    lifted-async-0.10.0.3 (lib)
Starting     enclosed-exceptions-1.0.3 (lib)
Building     enclosed-exceptions-1.0.3 (lib)
Installing   enclosed-exceptions-1.0.3 (lib)
Completed    enclosed-exceptions-1.0.3 (lib)
Starting     conduit-1.3.1 (lib)
Building     conduit-1.3.1 (lib)
Installing   conduit-1.3.1 (lib)
Completed    conduit-1.3.1 (lib)
Starting     x509-1.7.4 (lib)
Building     x509-1.7.4 (lib)
Installing   x509-1.7.4 (lib)
Completed    x509-1.7.4 (lib)
Starting     shelly-1.8.1 (lib)
Building     shelly-1.8.1 (lib)
Installing   shelly-1.8.1 (lib)
Completed    shelly-1.8.1 (lib)
Starting     conduit-extra-1.3.0 (lib)
Building     conduit-extra-1.3.0 (lib)
Installing   conduit-extra-1.3.0 (lib)
Completed    conduit-extra-1.3.0 (lib)
Starting     x509-store-1.6.6 (lib)
Building     x509-store-1.6.6 (lib)
Installing   x509-store-1.6.6 (lib)
Completed    x509-store-1.6.6 (lib)
Starting     x509-validation-1.6.10 (lib)
Building     x509-validation-1.6.10 (lib)
Installing   x509-validation-1.6.10 (lib)
Completed    x509-validation-1.6.10 (lib)
Starting     x509-system-1.6.6 (lib)
Building     x509-system-1.6.6 (lib)
Installing   x509-system-1.6.6 (lib)
Completed    x509-system-1.6.6 (lib)
Starting     tls-1.4.1 (lib)
Building     tls-1.4.1 (lib)
Installing   aeson-1.4.1.0 (lib)
Completed    aeson-1.4.1.0 (lib)
Installing   tls-1.4.1 (lib)
Completed    tls-1.4.1 (lib)
Starting     connection-0.2.8 (all, legacy fallback)
Building     connection-0.2.8 (all, legacy fallback)
Installing   connection-0.2.8 (all, legacy fallback)
Completed    connection-0.2.8 (all, legacy fallback)
Starting     http-client-tls-0.3.5.3 (lib)
Building     http-client-tls-0.3.5.3 (lib)
Installing   http-client-tls-0.3.5.3 (lib)
Completed    http-client-tls-0.3.5.3 (lib)
Starting     http-conduit-2.3.2 (lib)
Building     http-conduit-2.3.2 (lib)
Installing   http-conduit-2.3.2 (lib)
Completed    http-conduit-2.3.2 (lib)
Starting     cabal-src-0.3.0.2 (all, legacy fallback)
Building     cabal-src-0.3.0.2 (all, legacy fallback)
Symlinking cabal-src-install
Symlinking mega-sdist
```

**Comment by Magicloud** [comment]
`-v` appended to former command showed a lot more. But ended at Symlinking as well. No obvious errors.

cabal-install version 2.4.0.0
compiled using version 2.4.0.1 of the Cabal library

**Comment by hvr** [comment]
I can confirm; if you run with `-j1` the build failure becomes more apparent, and it's also noteworthy that the exit code is non-zero!

This is strongly related to the bug #5641 cc 

```
$ cabal v2-install cabal-src -j1 --overwrite=always
Resolving dependencies...
Build profile: -w ghc-8.4.4 -O1
In order, the following will be built (use -v for more details):
 - cabal-src-0.3.0.2 (exe:cabal-src-install, exe:mega-sdist) (requires build)
Configuring cabal-src-0.3.0.2...
Preprocessing executable 'mega-sdist' for cabal-src-0.3.0.2..
Building executable 'mega-sdist' for cabal-src-0.3.0.2..
[1 of 1] Compiling Main             ( mega-sdist.hs, dist/build/mega-sdist/mega-sdist-tmp/Main.o )

mega-sdist.hs:71:9: error:
    Variable not in scope: conduitManagerSettings :: ManagerSettings
   |
71 |         conduitManagerSettings
   |         ^^^^^^^^^^^^^^^^^^^^^^
Symlinking 'cabal-src-install'
Symlinking 'mega-sdist'

$ echo $?
1
```

cc @typedrat

**Comment by fgaz** [comment]
The dead symlink part will be fixed by #5653. Refer to #5641 for the error reporting.
